python/ray/autoscaler/_private/autoscaler.py

```diff
@@ -352,9 +363,15 @@ def target_num_workers(self):
             ideal_num_workers = max(
                 ideal_num_workers,
                 int(np.ceil(cores_desired / cores_per_worker)))
-
-        return min(self.config["max_workers"],
-                   max(self.config["min_workers"], ideal_num_workers))
+        min_node_types_workers = 0
+        if self.resource_demand_scheduler:
+            for node_type in self.available_node_types:
+                min_node_types_workers += \
+                    self.available_node_types[node_type].get("min_workers", 0)
+        return min(
```

Comment:
We don't need this given the above check to avoid deleting on idle right? Please revert. We also don't want to increase this value since it might trigger launch of default node type nodes.